romancal/source_catalog/source_catalog.py

```diff
@@ -162,7 +160,7 @@ def pixel_area(self):
         pixel_area = self.model.meta.photometry.pixelarea_steradians
         if pixel_area < 0:
             pixel_area = (self._pixel_scale**2).to(u.sr)
-        return pixel_area
+        return pixel_area.value
```

Comment:
This is fine, though I'm a little confused since I thought you removed the units from model.meta.photometry.